nautobot_ssot/migrations/0005_sync_compressed_diff.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.16 on 2023-05-23 20:05
+
+from django.db import migrations, models
+import nautobot_ssot.models
+
+
+def copy_diff_to_compressed_diff(apps, schema_editor):
+    for sync in nautobot_ssot.models.Sync.objects.all():
+        sync.compressed_diff = sync.diff
+        sync.validated_save()
+
+
+def undo_diff_copy(apps, schema_editor):
+    pass
```

Comment:
I think this gets pretty tricky once migration 0006 has been applied, unless we also crafted that migration file so that we could properly reverse both migrations.

Reply:
Actually, in an attempt to address the previous comment, I have restart migrations, and we will need to discuss these migrations further

Reply:
@glennmatthews @chadell I have tested the new migration files, and they are reversible, and will function as long as the commits are also rolled back. I also verified makemigrations does not pick up any missing migrations rolling forward and backward.